core/domain/classifier_services.py

```diff
@@ -213,12 +306,6 @@ def create_classifier(job_id, classifier_data):
         classifier.state_name, classifier.algorithm_id,
         classifier.classifier_data, classifier.data_schema_version)
 
-    # Create the mapping from <exp_id,exp_version,state_name>
-    # to classifier_id.
-    create_classifier_exploration_mapping(
-        classifier.exp_id, classifier.exp_version_when_created,
-        classifier.state_name, classifier_id)
-
     return classifier_id
```

Comment:
Thanks for the confirmation, @prasanna08! Separate PR sounds good to me. @pranavsid98, let's please just file a separate issue for this so we don't forget it.